### ISSUE TITLE
[PDFViewer] missing labels in tools buttons

### ISSUE THREAD

**Issue by iadnonov** [telerik/kendo-react#1986]
## I'm submitting a...

* Bug report 

## Current behavior

  * PDFViewer valuate axe DevTools missing labels for tools buttons

## Expected behavior

  * PDFViewer tools buttons should not valuate missing labels error

## Minimal reproduction of the problem with instructions
  1. Open the following demo page: https://www.telerik.com/kendo-react-ui/components/pdfviewer/toolbar/
  2. Make axe DevTools test of the component.

Ticket ID: 1640852

**Comment by iadnonov** [comment]
Fixed in dev version [v7.2.3-develop.1](https://www.telerik.com/kendo-react-ui-develop/components/changelogs/ui-for-react/#v7.2.3-develop.1)
